### DISCUSSION TITLE
Dropdown - item colour customisation

### DISCUSSION THREAD

**Discussion by cdpestitschek** [twilio-labs/paste#2601] [General] [answered]
Hi Paste team,

We are updating an existing component in our application Twilio Flex, and we would like to apply a different colour in one of the text elements inside the dropdown. (see example below)

Existing:
![image](https://user-images.githubusercontent.com/100429369/183722430-267223bf-a55e-444f-bbaf-af4ab0b306f7.png)

If we adapt to Paste as is:
<img width="235" alt="Paste way" src="https://user-images.githubusercontent.com/100429369/183722242-2976cc60-1d69-4e9c-b68f-f234eb8b97f1.png">

How we would like to propose:
![image](https://user-images.githubusercontent.com/100429369/183731654-f112a956-1838-47f7-b60e-df65724ec3a5.png)

I'm making this request based on tests that we carried out with users, and also understanding that we currently have this variations for buttons.

![image](https://user-images.githubusercontent.com/100429369/183722764-25263ae5-967c-4142-901b-4492943a1c90.png)

We took care and also validated the copy present to cover the point mentioned in the Paste guidlines.
> Don’t communicate with color alone. When choosing a destructive Button, make sure the Button text also indicates the action is destructive

Looking forward to hear from you,
Carol

**Comment by loreina** [comment]
Hi @cdpestitschek, we're working on adding a destructive variant for menu item in code and Figma. I'll reach back out once those are available shortly. Thanks for being patient!

**Comment by cdpestitschek** [comment]
Hi Loreina, thank you for answering

Glad to hear you're working on this variant. But could you please provide a timeframe/estimate?

**Reply by loreina** [reply]
Hey! I have a [Figma branch](https://www.figma.com/file/ocYkaWxTrLAwOI89pzuAZ8/feat(Menu)%3A-add-destructive-menu-item-variant?node-id=15080%3A99307) up for the changes, and a [PR in code](https://github.com/twilio-labs/paste/pull/2604). I'll follow up in this thread once they've been reviewed and merged in.

**Comment by loreina** [comment, marked as answer]
Hi @cdpestitschek, a new destructive hover variant for Menu item is [available in Figma](https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/Paste-Components?node-id=445%3A2033). We still have a PR #2604 up for review that I'm hopeful will be merged shortly. Thanks!

**Reply by SiTaggart** [reply]
Hi @cdpestitschek 

The new destructive menu item is now available in Code too.
